.github/ISSUE_TEMPLATE/eip-tracker.yaml

```
@@ -0,0 +1,79 @@
+name: "EIP Implementation Tracker"
+description: "Track specification and testing progress for an EIP"
+title: "EIP ${{ form.eip_number }} Progress Tracker"
+labels:
+  - A-spec-specs
+  - A-spec-tests
+  - C-eip
+  - C-test
+
+body:
+  - type: input
+    id: eip_number
+    attributes:
+      label: "EIP Number"
+      description: "Enter the EIP number (digits only)."
+      placeholder: "e.g., 8024"
+    validations:
+      required: true
+
+  - type: input
+    id: eip_title
+    attributes:
+      label: "EIP Title"
+      description: "Copy the title from the EIP."
+      placeholder: "e.g., Backwards compatible SWAPN, DUPN, EXCHANGE"
+    validations:
+      required: true
+  
+  - type: dropdown
+    id: fork
+    attributes:
+      label: "Fork"
+      description: |
+        Specify the target fork **only if the EIP has reached the CFI stage**.
+        More info: https://eips.ethereum.org/EIPS/eip-7723#considered-for-inclusion
+      options:
+        - TBD
+        - amsterdam
+        - bogota
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        ## EIP ${{ form.eip_number }}: ${{ form.eip_title }}
+
+        **Link:** https://eips.ethereum.org/EIPS/eip-${{ form.eip_number }}
+
+        ## Target Fork
+
+        Fork **${{ form.fork }}**
+
+        - [ ] Add issue to fork milestone (if applicable).
+
+        ## Ownership
+        
+        Owner(s): **TBD**
+
+        **Important note:** A specifications specialist and a testing specialist should ideally share ownership of the EIP.
+
+        ## Specification + Testing Status
+
+        - [ ] Testing complexity assessed and documented.
+        - [ ] Specification implementation merged to `eips/${{ form.fork }}/eip-${{ form.eip_number }}`.
+        - [ ] Specification updates merged to the corresponding `forks/${{ form.fork }}` branch.
+        - [ ] Required testing framework modifications implemented.
```

I think it's important to document when we, either on our own or as a team, have had to make decisions about the architecture of an implementation which are not obvious/straightforward and explain why those choices were made. I'm thinking of cases where we have structurally altered the implementation due to code review comments or there were multiple good options for implementation.

```suggestion
        - [ ] Specification implementation merged to `eips/${{ form.fork }}/eip-${{ form.eip_number }}`.
        - [ ] Specification updates merged to the corresponding `forks/${{ form.fork }}` branch.
        - [ ] Important architectural implementation choices documented
        - [ ] Required testing framework modifications implemented.
```

---

I paraphrased a bit:
> EIP updates proposed in case of architectural choices surfaced during implementation.

I used the word "proposed" here because we cannot guarantee a change goes into the EIP (technically, because most of the times the EIP process welcomes implementation feedback very well), just to not imply that we have to try to force this because it's outside of the EIP process.